screenpy/resolutions/contains_the_entry.py

```diff
@@ -71,6 +77,5 @@ def __init__(self, *kv_args: Any, **kv_kwargs: Any) -> None:
                     (kv_args[i], kv_args[i + 1]) for i in range(0, len(kv_args), 2)
                 ]
                 self.entries = dict(pairs, **kv_kwargs)
-        self.entries_to_log = ", ".join(
-            f"{{{k}: {v}}}" for k, v in self.entries.items()
-        )
+        self.entry_plural = "entries" if len(self.entries) != 1 else "entry"
+        self.entries_to_log = ", ".join(f"{k}->{v}" for k, v in self.entries.items())
```

Comment:
I also changed the output. This is what the `describe` method would say for `ContainsTheEntries(spam="eggs", tree="Larch")` before this change:
```quote
A mapping with the entries {spam: eggs}, {tree: larch}.
```

and after:
```quote
A mapping with the entries spam->eggs, tree->larch.
```

The string before looked weird to me, and i think would get especially annoying to look at if there was a large list of key->value mappings. I think this new way looks a little better. Do you have thoughts?

Reply:
The notation `X->Y` wouldn't have been my first thought to illustrate the key:value pairs.  What if we just show a single dictionary?  `{key1: val1, key2:val2, ...}`

Reply:
I worried that it could be easily mistaken for an equality test rather than a "does the mapping have these entries" test. Also, it turns out writing that expected string in the test is pretty annoying! Haha.

I'm not sure when/where i started writing `key->value`, but i've been doing it for a while now. Maybe just like this?
```quote
A mapping with the entries spam: eggs, tree: larch.
```